src/brevitas/export/onnx/handler.py

```diff
@@ -88,6 +88,38 @@ def kernel_shape(module):
             return list(module.kernel_size)
 
 
+class Kernel3dApplHandlerMixin(ABC):
+
+    @staticmethod
+    def padding(module):
+        if isinstance(module.padding, int):
+            padding = [module.padding] * 6
+        else:
+            padding = list(module.padding) + list(module.padding)
+        return padding
+
+    @staticmethod
+    def stride(module):
+        if isinstance(module.stride, int):
+            return [module.stride] * 4
```

Comment:
I think this should actually be `return [module.stride] * 3`

Reply:
ah - I thought they were going up by 2 - fixing now